plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/glue/TestDeltaLakeGlueMetastore.java

```diff
@@ -116,6 +117,7 @@ public void setUp()
                 // connector modules
                 new DeltaLakeMetastoreModule(),
                 new DeltaLakeModule(),
```

Comment:
This may be related to https://github.com/trinodb/trino/commit/2cb84fe52454766558c9b6c2e9dfea6a4d7a5bc7
cc @findepi